tests/load_main_menu_test.py

```
@@ -0,0 +1,136 @@
+    context = browser.new_context(
+        viewport={"width": 1280, "height": 720},
+        record_har_path="artifacts/har.har"  # Optional network trace
+    )
+    page = context.new_page()
+    # CDP for V8 coverage
+    cdp_session = None  # Initialize to None outside try
+    try:
+        cdp_session = context.new_cdp_session(page)
+        cdp_session.send("Profiler.enable")
+        cdp_session.send("Profiler.startPreciseCoverage", {"callCount": True, "detailed": True})
+    except:
```

Ruff (0.14.11)

Do not use bare `except`

(E722)